apps/fluent-tester/src/TestComponents/Menu/MenuRefs.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ButtonV1 as Button } from '@fluentui/react-native';
+import { Menu, MenuItem, MenuTrigger, MenuPopover, MenuList } from '@fluentui-react-native/menu';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+
+export const MenuTriggerChildRef: React.FunctionComponent = () => {
+  const ref = React.useRef(null);
+  const onClick = React.useCallback(() => {
+    ref?.current?.focus();
+  }, [ref]);
+
+  return (
+    <Stack style={stackStyle}>
+      <Button onClick={onClick}>Click to focus Menu trigger</Button>
```

Comment:
When "Click to focus Menu trigger" button is clicked, should the menu item or the other button get focus? This is what I'm seeing on Mac:

https://user-images.githubusercontent.com/67026167/201215603-fa4cbe68-7d97-4e2c-bcc6-eb73a11cd0b8.mov

Reply:
Is this because we have the `onClickWithFocus` hook and the timing of that on macOS and win32 is different? AKA focus is going from MenuTrigger -> Button on macOS, but Button -> MenuTrigger on macOS ?

Reply:
@Saadnajmi yup, I opted out the focus ref in the `useOnPressWithFocus` hook and that fixed it

Reply:
Now my question is why this is a macOS specific issue

Reply:
It's not - it just flickers on win32 enough that I know that the ref.current.focus() is working. I couldn't really think of a better way to test it.

Reply:
We have a backlog item to actually _not_ use that hook because I think native macOS controls should actually not be "stealing" focus on click most of the time. I interpret this as "The test was buggy to begin with" so it's kind of a moot point / out of scope. I'm ok with leaving it in the PR since I don't know how to test it either...